### ISSUE TITLE
Add secondary options for nested blocks to the `block-closing-brace-empty-line-before` rule

### ISSUE THREAD

**Issue by gian1200** [stylelint-stylistic/stylelint-stylistic#60]
### What is the problem you're trying to solve?

If:
- "@stylistic/max-empty-lines" is set to 1, and 
- "@stylistic/block-closing-brace-newline-after" to "always"

it is not possible to say "except to last block" or "last-nested". This could be useful with the last block inside nested blocks (eg. Media Queries)

### What solution would you like to see?

Add optional secondary options to `block-closing-brace-space-after`. Something like `"except": ["last-nested"]` and/or `"ignore": ["last-block"]`

**Comment by MorevM** [comment]
Good day @gian1200,

Can you provide some CSS examples that trigger the problem?
A comparison between the current and desired formatting would also help.

I'm not sure what we're trying to achieve - your issue mentions two rules, [`block-closing-brace-newline-after`](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/lib/rules/block-closing-brace-newline-after/README.md) and [`block-closing-brace-space-after`](https://github.com/stylelint-stylistic/stylelint-stylistic/tree/main/lib/rules/block-closing-brace-space-after), but I wasn't able to reproduce any formatting that seemed incorrect :)

**Comment by gian1200** [comment]
Thanks for the response`, @MorevM . I'll try to rephrase it:

There is `rule-empty-line-before` rule, which helps check first line on a nested block. However, there is no similar rule for the end of the block

```css
@media (width >= 1px) {

	.dummy-class {
		font-weight: 400;
	}

	.dummy-class li {
		display: inline-block;
	}

}

@media (width >= 1px) {
	.dummy-class { /* line 14 */
		font-weight: 400;
	}

	.dummy-class li {
		display: inline-block;
	} /* line 20 */
} /* line 21 */
```

 2 errors are desired, but only 1 is thrown:

` 14:2  ✖  Expected empty line before rule  rule-empty-line-before`

My feature request is to have a rule that also throws an error at line 20 or 21.

**Comment by MorevM** [comment]
@gian1200 

Just to confirm: you expect this

```scss
@media (width >= 1px) {
  .foo {
    color: red;

    a {
      color: blue;
    }
  }
}
```

to trigger an error and be autofixed into this:

```scss
@media (width >= 1px) {

  .foo {
    color: red;

    a {
      color: blue;
    }
    
  }

}
```

Is that right?

**Comment by gian1200** [comment]
I didn't thought that far, but yes (depending on configuration). Basically, to have the equivalent of `rule-empty-line-before`, for closing bracket.

In your example, I `"rule-empty-line-before": ["always"]` is assumed, right?

Ideally, it should also be possible to have the equivalent of `"rule-empty-line-before": ["never"]`  for the closing bracket.

PS: In the documentation there are these secondary options "inside-block", "first-nested". Maybe there could be also their equivalent ("inside-block", "last-nested")

https://stylelint.io/user-guide/rules/rule-empty-line-before/
